src/api/public/apidocs-new/components/schemas/distribution_hash.yaml

```diff
@@ -0,0 +1,42 @@
+type: object
+properties:
+  id:
+    type: integer
```

Comment:
Right now if you request a distribution with an id e.g. /distributions/16626 you get a response that includes the following:  
```<id type="integer">16626</id>```
Adding this `type` attribute however seems to be impossible with OpenAPI as it doesn't allow integers to have xml attributes like an object. And an object can't have a value. As @eduardoj pointed out this `type` attribute is unnecessary anyway as the API doesn't really need to specify the types.

Reply:
This is the case because we 'just' render the pure object in the controller. Instead of writing a view.